custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -775,24 +776,70 @@ def _run_inside_docker(self, options, run_mode, raw_arguments):
         return retcode
 
     def _maybe_build_image(self, docker_image_or_directory):
-        ret_docker_image = None
+        def _get_requirements_lines(reqs_file_path):
+            if not os.path.exists(reqs_file_path):
+                return None
+
+            with open(reqs_file_path) as f:
+                lines = f.readlines()
+                lines = [l.strip() for l in lines]
+            return lines
 
+        ret_docker_image = None
         if os.path.isdir(docker_image_or_directory):
             docker_image_or_directory = os.path.abspath(docker_image_or_directory)
+            # Set image tag to the dirname of the docker context.
+            # This may be confusing if try to build different images from different contexts,
```

Comment:
I think this will be confusing to people, maybe have like two dirnames?

Reply:
Agreed.
DONE.